R/loggers.R

```diff
@@ -0,0 +1,111 @@
+#' App Logging
+#' 
+#' `fiery` has a build in logging mechanism that lets you capture event 
+#' information however you like. Every user-injested warnings and errors are
+#' automatically captured by the logger along with most system errors as well.
+#' `fiery` tries very hard not to break due to faulty app logic. This means that
+#' any event handler error will be converted to an error log without `fiery` 
+#' stopping. In the case of request handlers a 500L response will be send back
+#' if any error is encountered.
+#' 
```

Comment:
Does this really cover *any* error/warning? It seems as though this error still isn't reported https://github.com/thomasp85/routr/issues/5

Reply:
This is because routr catches all errors so it doesn't bubble up to the fiery layer - I need to think about how the two interacts...